contracts/clients/package.json

```diff
@@ -1,33 +1,20 @@
 {
-  "name": "bls-wallet-signer",
-  "version": "0.8.0-rc.1",
-  "description": "Client-side tool for signing bls transaction data",
+  "name": "bls-wallet-clients",
+  "version": "0.4.0",
+  "description": "Client libraries for interacting with BLS Wallet components",
   "main": "dist/src/index.js",
   "types": "dist/src/index.d.ts",
+  "repository": "https://github.com/jzaki/bls-wallet/tree/main/client",
+  "author": "Andrew Morris",
+  "license": "MIT",
+  "private": false,
   "scripts": {
-    "clean": "rm -rf dist",
     "build": "tsc",
     "watch": "tsc -w",
     "test": "mocha dist/**/*.test.js"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/jzaki/bls-wallet-signer.git"
-  },
-  "keywords": [
-    "ethereum",
-    "optimism",
-    "bls",
-    "signature",
-    "aggregation"
-  ],
-  "author": "Andrew Morris",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/jzaki/bls-wallet-signer/issues"
-  },
-  "homepage": "https://github.com/jzaki/bls-wallet-signer#readme",
```

Comment:
Some of these might be worth keeping to make package more discoverable, but can always add back in later.